client/src/modules/home/home.js

```diff
@@ -15,10 +15,9 @@ class HomeController {
     this.name = 'Your Plans';
 
     this.deletePlan = (planId) => {
```

Comment:
class functions should be implemented as methods on the class, not declared in the constructor unless necessary